src/control/lib/netdetect/netdetect_test.go

```diff
@@ -173,9 +173,10 @@ func TestValidateNetworkConfig(t *testing.T) {
 // TestDeviceAliasErrors uses XML topology data to simulate real systems.
 // hwloc will use this topology for queries instead of the local system
 // running the test.
-// This test verifies that GetDeviceAlias() handles the error cases when a specified
+// This test verifies that getDeviceAliasWithSystemList() handles the error cases when a specified
 // device cannot be matched to a sibling.
 func TestDeviceAliasErrors(t *testing.T) {
+	mockSystemDevices := []string{}
```

Comment:
Just a nit - I'd just pass this empty slice directly down below in the test function. It's always empty and doesn't really represent any devices. No need to change it unless you need to add something else for this PR.

Reply:
Sure, I can fix that.